### DISCUSSION TITLE
Using this library in Blazor (WebAssembly)

### DISCUSSION THREAD

**Discussion by unarsu** [micjahn/ZXing.Net#358] [Q&A]
Do you think it's feasible to use this library in Blazor to accomplish the following: mobile clients (think Android, iOS/ipadOS, Windows ARM) using the camera to have scan the barcode (I need UPC-A, Code 39, Aztec and QR)?

I've tried some of the other libraries that port ZXing via GWT or Emscripten to JavaScript, but they are all a bit flaky and difficult for me to fix. Something that was in .NET and using Blazor, would be far easier for me to maintain.

Would appreciate any thoughts. Thanks!

**Comment by micjahn** [comment]
You should take a look at this issue here and the mentioned links
https://github.com/micjahn/ZXing.Net/issues/366
